vmm_tests/vmm_tests/tests/tests/x86_64/openhcl_linux_direct.rs

```diff
@@ -406,6 +406,13 @@ async fn storvsp(config: PetriVmBuilder<OpenVmmPetriBackend>) -> Result<(), anyh
     // Account for the pipette drive too
     assert_eq!(reported_sizes.len(), 3);
 
+    let _io_to_sda = cmd!(sh, "sh -c 'dd if=/dev/urandom of=/dev/sda bs=1M count=100'")
+        .run()
+        .await?;
+    let _io_to_sdb = cmd!(sh, "sh -c 'dd if=/dev/urandom of=/dev/sdb bs=1M count=100'")
+        .run()
+        .await?;
+
```

Comment:
I imagine this is too naive because of the pipette drive. I'll iterate when I get a moment.